File: app/src/main/java/com/aldringames/scjd/sbsp/common/BaseActivity.java
package com.aldringames.scjd.sbsp.common;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class BaseActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
	super.onCreate(savedInstanceState);
	overridePendingTransition(0, 0);
    }
}

File: app/src/main/java/com/aldringames/scjd/sbsp/common/BaseActivity.kt
package com.aldringames.scjd.sbsp.common

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class BaseActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
	super.onCreate(savedInstanceState)
	overridePendingTransition(0, 0)
    }
}
